test/styles/RRR.R

```diff
@@ -39,38 +48,110 @@ fun_call(argument1,
 fun_call(
     argument1,
     argument2
-    )
+)
 
 ## 3
 fun_call(parameter = (
     stuff
-    ),
-         argument)
+),
+argument)
 
 ## 4
 fun_call(parameter = fun_argument(
-             stuff
-    ),
-         argument)
+    argument1
```

Comment:
> But it wouldn't be hard to make the pushing behaviour a setting if you want.

I thought that you already have a setting to indent arguments of the inner call.